aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -261,6 +270,71 @@ public static CreateDbInstanceRequest createDbInstanceRequest(
                 .build();
     }
 
+    static Instant stringToInstant(String restoreTimeString) {
+        if (StringUtils.isNotBlank(restoreTimeString)) {
+            try {
+                return LocalDateTime.parse(
+                                restoreTimeString,
+                                DateTimeFormatter.ofPattern(RESTORE_TIME_PATTERN)
+                        )
+                        .atOffset(ZoneOffset.ofHours(0))
```

Comment:
Offset can be built-in into timestamp, e.g. `2007-04-05T12:30−02:00` or `2007-04-05T12:30+05:30`